config/controllers/hive_controllers_role.yaml

```diff
@@ -49,6 +49,7 @@ rules:
   - ""
   resources:
   - pods
+  - pods/log
```

Comment:
Was this one intentional?

Reply:
Yes, had added a note to the PR description..

---
**NOTE**: the controller can only pass on the permissions that it has i.e. the current hive-cluster-pool-admin cluster role is a subset of the hive-controller's permission. If the controller permissions are less that what the hive-cluster-pool-admin cluster role, then when creating a rolebinding in the clusterdeployment namespace we see an error like 
```
time="2020-11-30T21:21:11.129Z" level=error msg="could not create rolebinding" clusterPool=default/adahiya-pool-1 controller=clusterpool error="rolebindings.rbac.authorization.k8s.io \"hive-cluster-pool-admin-binding\" is forbidden: user \"system:serviceaccount:hive:hive-controllers\" (groups=[\"system:serviceaccounts\" \"system:serviceaccounts:hive\" \"system:authenticated\"]) is attempting to grant RBAC permissions not currently held:\n{APIGroups:[\"\"], Resources:[\"pods/log\"], Verbs:[\"get\" \"list\" \"watch\"]}\n{APIGroups:[\"apiextensions.k8s.io\"], Resources:[\"customresourcedefinitions\"], Verbs:[\"get\" \"list\" \"watch\"]}" reconcileID=gn7pt9sd rolebinding=adahiya-pool-1-kpkbg/hive-cluster-pool-admin-binding
```

Reply:
Ah yes! Thanks.